pkg/controller/gitopsservice/gitopsservice_controller.go

```diff
@@ -213,6 +217,60 @@ func (r *ReconcileGitopsService) Reconcile(request reconcile.Request) (reconcile
 		}
 	}
 
+	// Define Service account for backend Service
+	serviceAccountObj := newServiceAccount(serviceNamespacedName)
```

Comment:
New resources are not watched by the operator. It didn't reconcile back when the resources were updated/deleted.

Reply:
yeah, I have seen a similar behaviour for other resources created by the operator. if I manually update the gitops backend service deployment, it doesn't reconcile either !! Maybe we can open a bug and handle this in a separate PR?

@chetan-rns, What do you suggest ?

Reply:
The new resource should be watched.   

It looks like backend deployment/service are watched.  
https://github.com/redhat-developer/gitops-operator/blob/master/pkg/controller/gitopsservice/gitopsservice_controller.go#L101-L115

Reply:
The events are filtered by this predicate. https://github.com/redhat-developer/gitops-operator/blob/697a419b67b30cc1fcb81bd0db716ce17c8262f6/pkg/controller/gitopsservice/gitopsservice_controller.go#L69

Reply:
ohh probably !! 
I'll fix the watch part of it ! Thanks @chetan-rns @wtam2018